tests/integration/local/start_api/test_start_api.py

```diff
@@ -664,6 +664,30 @@ def test_patch_call_with_path_setup_with_any_swagger(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), {"hello": "world"})
 
+    @pytest.mark.flaky(reruns=3)
+    @pytest.mark.timeout(timeout=600, method="thread")
+    def test_http_api_payload_v1_should_not_have_operation_id(self):
+        response = requests.get(self.url + "/httpapi-operation-id-v1", timeout=300)
+        self.assertEqual(response.status_code, 200)
+
+        response_data = response.json()
+        self.assertEqual(response_data.get("version", {}), "1.0")
+        # operationName or operationId shouldn't be processed by Httpapi swaggers
+        self.assertIsNone(response_data.get("requestContext", {}).get("operationName"))
+        self.assertIsNone(response_data.get("requestContext", {}).get("operationId"))
+
+    @pytest.mark.flaky(reruns=3)
+    @pytest.mark.timeout(timeout=600, method="thread")
+    def test_http_api_payload_v2_should_not_have_operation_id(self):
+        response = requests.get(self.url + "/httpapi-operation-id-v2", timeout=300)
+        self.assertEqual(response.status_code, 200)
+
+        response_data = response.json()
+        self.assertEqual(response_data.get("version", {}), "2.0")
+        # operationName or operationId shouldn't be processed by Httpapi swaggers
+        self.assertIsNone(response_data.get("requestContext", {}).get("operationName"))
```

Comment:
How do you know `operationName` does not exist vs `operationName` is in the dict but with no value (`None`)?

Reply:
Good catch! I updated the tests here and other places.